integration-tests/src/test/java/org/web3j/protocol/besu/BesuOnChainPrivacyIntegrationTest.java

```diff
@@ -37,6 +39,7 @@
 import static org.junit.jupiter.api.Assertions.*;
 
 @Disabled
```

Comment:
Should this be kept disabled?

Reply:
yes the method is not enabled in the besu container